themes/jb/layouts/partials/meta.html

```diff
@@ -1,9 +1,9 @@
 <!-- Meta Ttitles -->
-<title itemprop="name">{{ .Title }} | {{ .Site.Title }}</title>
-<meta property="og:title" content="{{ .Title }} | {{ .Site.Title }}" />
-<meta name="twitter:title" content="{{ .Title }} | {{ .Site.Title }}" />
-<meta itemprop="name" content="{{ .Title }} | {{ .Site.Title }}" />
-<meta name="application-name" content="{{ .Title }} | {{ .Site.Title }}" />
+<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}</title>
```

Comment:
Just a note for the future @ChanceM , another solution that's a bit more DRY for hugo is to use the [with function](https://gohugo.io/functions/with/).

While technically both accomplish what you're trying for, when I initially looked at this I was thinking "what is he trying to do...🤔". Because technically you're still accessing `.Title` even if the `.Title` if condition fails right now, because it's outside of the `if` statement. Hugo is inserting a blank space and HTML doesn't really care about that.

So, it's a larger [gulf of evaluation](https://en.wikipedia.org/wiki/Gulf_of_evaluation) (heard about this phrase recently and wanted to use it somewhere 😅) to understand why your solution works IMO.

If you'd used `with` you could've written it like this:
```suggestion
<title itemprop="name">{{ with .Title -}}{{- . }} | {{end}}{{ .Site.Title }}</title>
```

Reply:
:+1:  Definitely still learning with Hugo, I appreciate the feedback!